cluster-up/cluster/kind/common.sh

```diff
@@ -228,6 +228,7 @@ function _add_worker_extra_mounts() {
   extraMounts:
   - containerPath: /var/log/audit
     hostPath: /var/log/audit
+    readOnly: false
```

Comment:
`true` :)

Reply:
Sorry my brain is locked I guess :facepalm:, fixed